src/chapters/data-and-variables/values-and-data-types.rst

```diff
@@ -92,35 +103,57 @@ JavaScript doesn't care whether you use single or double quotes to surround your
 More On Numbers
 ---------------
 
-When you type a large integer, you might be tempted to use commas between groups of three digits, as in ``42,000``. This is not a legal integer in JS, but it does mean something else, which is legal:
+When you type a large integer value, you might be tempted to use commas between groups of three digits, as in ``42,000``. This is not a legal integer in JavaScript, but it does mean something else, which is legal:
 
-.. code-block:: js
+.. sourcecode:: js
 
     console.log(42000);
     console.log(42,000);
 
+**Output**
+
+::
+
+   42000
+   42 0
 
-Well, that's not what we expected at all! Because of the comma, JS chose to treat this as a *pair* of values. In fact, the ``console.log()`` function can print any number of values as long as you separate them by commas. Notice that the values are separated by spaces when they are displayed.
+Well, that's not what we expected at all! Because of the comma, JavaScript chose to treat ``42,000`` as a *pair* of values. In fact, the ``console.log()`` function can print any number of values as long as you separate them by commas. Notice that the values are separated by spaces when they are displayed.
 
-.. code-block:: js
+.. sourcecode:: js
 
     console.log(42, 17, 56, 34, 11, 4.35, 32);
     console.log(3.4, "hello", 45);
 
-Remember not to put commas or spaces in your integers, no matter how big they are. Also revisit what we said in the previous chapter: formal languages are strict, the notation is concise, and even the smallest change might mean something quite different from what you intended.
+**Output**
+
+::
+
+   42 17 56 34 11 4.35 32
+   3.4 'hello' 45
+
+Remember not to put commas or spaces in your integers, no matter how big they are. Also revisit what we said in the previous chapter: formal languages are strict, the notation is concise, and even the smallest change might mean something quite different from what you intend.
 
 Type Systems
 ------------
 
 .. index:: ! type system
 
-Every programming langauge has a **type system**, which is the set of rules that determine how it deals with data of different types. In particular, how values are divided up into different data types of one characteristic of a type system.
+Every programming langauge has a **type system**, which is the set of rules that determine how it deals with data of different types. In particular, how values are divided up into different data types is one characteristic of a type system.
```

Comment:
Language is misspelled.

Reply:
Good catch on the of/is change.  In addition, how about a rephrasing of the first sentence?  Something like, "Every programming language has a **type system**, which is a set of rules that determine how to deal with different data types."  This would remove the ambiguous 'it' from the sentence.

Reply:
I removed the "it" pronoun by explicitly referring to "the language."